src/json/members/jessez.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "Jesse Zong",
+    "pennkey": "jessez",
+    "major": "Computer Science",
+    "school": "Engineering",
+    "bio": "Hi, I'm Jesse! I'm a CIS major in engineering",
+    "hometown": "Ellicott City, Maryland",
+    "team": "Penn Mobile",
+    "roles": ["Backend Engineer"],
+    "photo": "https://i.imgur.com/1p5qwK5.jpg",
+    "linkedin": "https://www.linkedin.com/in/jesse-zong-607199227/",
+    "website": "",
+    "github": "https://github.com/dr-Jess",
+    "semester_joined": "2023F",
```

Comment:
See onboarding guide for semester notation: https://www.notion.so/pennlabs/Creating-or-Updating-a-Member-Page-f896d85568b848c594e005708f8cb1a6

Reply:
fixed